app/src/androidTest/java/com/github/epfl/meili/auth/GoogleSignInActivityTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.github.epfl.meili.auth
+
+import android.app.Instrumentation
+import android.content.Intent
+import android.location.LocationManager
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intending
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.internal.runner.junit4.statement.UiThreadStatement.runOnUiThread
+import com.github.epfl.meili.R
+import com.github.epfl.meili.database.FirestoreDatabase
+import com.github.epfl.meili.map.MapActivity
+import com.github.epfl.meili.util.LocationService
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.ListenerRegistration
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+
+
+@RunWith(AndroidJUnit4::class)
+class GoogleSignInActivityTest {
```

Comment:
Why is this file being added?

Reply:
Because I accidentally deleted it in another PR so here it is coming back